packages/SwingSet/src/kernel/vatManager/deliver.js

```diff
@@ -1,5 +1,14 @@
-import { assert, details as X } from '@agoric/assert';
-import { insistMessage } from '../../message';
+/** @type { (delivery: VatDeliveryObject, prefix: string) => string } */
+function summarizeDelivery(vatDeliveryObject, prefix = 'vat') {
```

Comment:
strange name for something that always returns a `... failed` message.

Reply:
it's really a "make a message to log just in case it fails" function, but that name seemed expedient.

Reply:
`failureMessage`?

Reply:
`summarizeDeliveryFailure` maybe?